test/core/br_table.wast

```diff
@@ -1552,6 +1562,35 @@
   "type mismatch"
 )
 
+(assert_invalid
+  (module
+    (func (param i32) (result i32)
+      (loop (result i32)
+        (block (result i32)
+          i32.const 1
+          local.get 0
+          br_table 0 1
```

Comment:
Nit: use folded form here as rest of file?

Reply:
Changed.